packages/jest-test-result/src/index.ts

```diff
@@ -22,4 +22,5 @@ export {
   Status,
   Suite,
   TestResult,
+  Callsite,
```

Comment:
Alphabetical

Reply:
We need to add linter rule for this. What do you think?

Reply:
yeah sure. we already have `sort-keys`, kinda odd it doesn't cover `import`/`export`

Reply:
Yea, there is no sorting rules for exports yet

Reply:
If there's a lint rule for this, a PR adding it would be very welcome!

Reply:
If there's a lint rule for this, a PR adding it would be very welcome!